### PR TITLE
Home Screen quick actions

### PR DESCRIPTION
**Pull request by bozidarsevo**
## Description

Added [home screen quick actions](https://developer.apple.com/documentation/uikit/menus_and_shortcuts/add_home_screen_quick_actions):
- New Order
- Orders
- Add a product
- Collect Payment

## Testing instructions

- press and hold the Woo icon on the home screen
- you should get 4 actions, as described above
- Test all by tapping them and make sure that each one does what it says

## Screenshots

Simulator Screenshot - iPhone 15 Pro - 2024-03-20 at 10 10 22

| New Order      | Orders  |
| ----------- | ----------- |
| Simulator Screen Recording - iPhone 15 Pro - 2024-03-20 at 10 13 04      | Simulator Screen Recording - iPhone 15 Pro - 2024-03-20 at 10 13 16       |

| Add a product      | Collect Payment  |
| ----------- | ----------- |
| Simulator Screen Recording - iPhone 15 Pro - 2024-03-20 at 10 13 27 | Simulator Screen Recording - iPhone 15 Pro - 2024-03-20 at 10 13 38 |

---
- [ ] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

### CODE REVIEW

**Comment by dangermattic** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Message">
          1 Message
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:book:</td>
      <td data-sticky="false">This PR is still a Draft: some checks will be skipped.</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_pr-check">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

**Comment by dangermattic** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">This PR is assigned to the milestone <a href="https://github.com/woocommerce/woocommerce-ios/milestone/190">17.8</a>. This milestone is due in less than 2 days.<br />
Please make sure to get it merged by then or assign it to a milestone with a later deadline.</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_pr-check">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

**Comment by dangermattic** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">This PR is assigned to the milestone <a href="https://github.com/woocommerce/woocommerce-ios/milestone/191">17.9</a>. This milestone is due in less than 2 days.<br />
Please make sure to get it merged by then or assign it to a milestone with a later deadline.</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_pr-check">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

File: RELEASE-NOTES.txt [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 17.9
 -----
+- Added 4 home screen quick actions: "New Order", "Orders", "Add a product", "Collect Payment" [https://github.com/woocommerce/woocommerce-ios/pull/12264/]
```
Comment (pmusolino) [inline RELEASE-NOTES.txt]: minor nit: i suggest to follow this format `[<priority indicator, more stars = higher priority>] <description> [<PR URL>]`. In this specific case, just the priority from 1 to 3 `*` here is missing.

**Review by pmusolino — APPROVED** [review]
Looks good to me! I tested it also on iPad, and it works like a charm. :shipit:

**Pull request merged**